extension-manifest-v3/src/utils/trackerdb.js

```diff
@@ -103,6 +103,8 @@ export function isCategoryBlockedByDefault(categoryId) {
     case 'adult_advertising':
     case 'email':
     case 'site_analytics':
+    case 'unidentified':
```

Comment:
I am not sure, but I think we removed that category, and everything, which does not fit its set to `misc`

Reply:
the extension code sets the `unidentified` category for unknown trackers